blocks/shared-styles/_children/button/index.story.jsx

```diff
@@ -9,77 +9,177 @@ export default {
   />
 );
 
-export const UserIconWithLabelOutline = () => (
+export const PrimaryReverseDualIcon = () => (
   <Button
-    buttonStyle={BUTTON_STYLES.OUTLINED}
-    buttonSize={BUTTON_SIZES.LARGE}
+    buttonStyle={BUTTON_STYLES.PRIMARY_REVERSE}
+    buttonType={BUTTON_TYPES.LABEL_AND_TWO_ICONS}
+    iconType="user"
     text="Sign up"
+    secondaryIconType="chevron-up"
+  />
+);
+// end primary reverse
+
+// start secondary
+export const SecondaryLabel = () => (
+  <Button
+    buttonStyle={BUTTON_STYLES.SECONDARY}
+    buttonType={BUTTON_TYPES.LABEL_ONLY}
+    text="Sign up"
+  />
+);
+
+export const SecondaryIconAndLabel = () => (
+  <Button
+    buttonStyle={BUTTON_STYLES.SECONDARY}
+    buttonType={BUTTON_TYPES.LABEL_AND_ICON}
     iconType="user"
+    text="Sign up"
   />
 );
 
-export const IconOnlyOutlined = () => (
+export const SecondaryIcon = () => (
   <Button
+    buttonStyle={BUTTON_STYLES.SECONDARY}
     buttonType={BUTTON_TYPES.ICON_ONLY}
-    text="Sign up"
     iconType="user"
-    buttonStyle={BUTTON_STYLES.OUTLINED}
+    ariaLabel="Sign up"
   />
 );
 
-export const IconWithLabel = () => (
+export const SecondaryDualIcon = () => (
   <Button
+    buttonStyle={BUTTON_STYLES.SECONDARY}
+    buttonType={BUTTON_TYPES.LABEL_AND_TWO_ICONS}
+    iconType="user"
+    text="Sign up"
+    secondaryIconType="chevron-up"
+  />
+);
+// end secondary
+
+// start secondary reverse
+export const SecondaryReverseLabel = () => (
+  <Button
+    buttonStyle={BUTTON_STYLES.SECONDARY_REVERSE}
+    buttonType={BUTTON_TYPES.LABEL_ONLY}
+    text="Sign up"
+  />
+);
+
+export const SecondaryReverseIconAndLabel = () => (
+  <Button
+    buttonStyle={BUTTON_STYLES.SECONDARY_REVERSE}
     buttonType={BUTTON_TYPES.LABEL_AND_ICON}
+    iconType="user"
     text="Sign up"
+  />
+);
+
+export const SecondaryReverseIcon = () => (
+  <Button
+    buttonStyle={BUTTON_STYLES.SECONDARY_REVERSE}
+    buttonType={BUTTON_TYPES.ICON_ONLY}
     iconType="user"
-    buttonStyle={BUTTON_STYLES.OUTLINED}
+    ariaLabel="Sign up"
   />
 );
 
-export const LabelOnly = () => (
+export const SecondaryReverseDualIcon = () => (
   <Button
-    buttonType={BUTTON_TYPES.LABEL_ONLY}
+    buttonStyle={BUTTON_STYLES.SECONDARY_REVERSE}
+    buttonType={BUTTON_TYPES.LABEL_AND_TWO_ICONS}
+    iconType="user"
     text="Sign up"
+    secondaryIconType="chevron-up"
+  />
+);
+// end secondary reverse
+
+export const PrimarySmall = () => (
+  <Button
+    buttonStyle={BUTTON_STYLES.PRIMARY}
+    buttonSize={BUTTON_SIZES.SMALL}
+    text="Sign up"
+  />
+);
+
+export const PrimaryLarge = () => (
+  <Button
+    buttonStyle={BUTTON_STYLES.PRIMARY}
+    buttonSize={BUTTON_SIZES.LARGE}
+    text="Sign up"
+  />
+);
+
+export const LongText = () => (
+  <Button
+    text="Sign up to get the very best experience and this other really long work and such and more and even more text.Sign up to get the very best experience and this other really long work and such and more and even more text."
```

Comment:
While this is possible - should we advocate for it? This would be valid in an `a` link button style but not an action `button`

Reply:
I can remove it

Reply:
updated